sdk/include/reactos/wine/test.h

```diff
@@ -548,25 +554,25 @@ void winetest_wait_child_process( HANDLE process )
     }
 }
 
-void winetest_disable_success_count_start()
+void winetest_nocount_start()
 {
-    get_tls_data()->nocount_level <<= 1;
+    get_tls_data()->nocount_level <<= 2;
```

Comment:
Why is this a bit shift instead of just adding / subtracting?

Reply:
It was a workaround for making this work with a for loop. The loop exit condition is checked before the code in question is run. With a simple counter you cannot check, whether you already ran the code once. With the shift you have different states: 1. "initialized" (which shifts the bits and sets the lowest 2 bits to 0, 2. "set", which sets the disable bit as intended and tells the loop condition to abort next time, and 3. "reset", which is back to where it was before the loop. After adding a second bit, I introduced a bug though: if one bit is already set and then the other bit is added, the previous bit is lost. I'll gix that.

Reply:
add a comment? (nobody knew about that idea except you :P)

Reply:
ok :)

Reply:
I changed it to use the same structure as todo_wine, this should make things better understandable. I also added a comment.